src/containers/pages/FocusInvestigation/map/active/tests/index.test.tsx

```diff
@@ -352,4 +362,48 @@ describe('containers/pages/FocusInvestigation/activeMap', () => {
     expect(currentGoalMock).not.toBeCalled();
     expect(FCMock).not.toBeCalled();
   });
+
+  it('renders the GisidaWrapper with structures', async () => {
+    const mock: any = jest.fn();
+    const supersetServiceMock: any = jest.fn(async () => []);
+
+    store.dispatch(fetchGoals([fixtures.goal3]));
+    store.dispatch(fetchJurisdictions([fixtures.jurisdictions[0]]));
+    store.dispatch(fetchPlans([fixtures.plan1]));
+    store.dispatch(fetchTasks(fixtures.tasks));
+    store.dispatch(setStructures(structures as structureDucks.Structure[]));
+
+    // check that we can get a plan from the store
+    expect(store.getState().plans.plansById[fixtures.plan1.id]).toEqual(fixtures.plan1); // passes
+    expect(getPlanById(store.getState(), fixtures.plan1.id)).toEqual(fixtures.plan1); // fails
```

Comment:
@moshthepitt @p-netm @kelvin-muchiri @ciremusyoka 

Any idea why this test is failing to get a plan from the store usig the `getPlanById()` method?

Reply:
Somehow `getPlanById` is returning `undefined`, which I'm not even sure how that's possible (since it should fall back to `null`). Also, when I copy the function itself into the test it works, but not when it's imported.

<img width="817" alt="Screen Shot 2020-02-24 at 8 03 14 AM" src="https://user-images.githubusercontent.com/4141527/75169022-b0f71c80-56dc-11ea-9fc2-f82201d1292d.png">

Reply:
@cKellyDesign Have noticed when you comment out the other tests the test is passing. My guess is some tests above are affecting the test but have not been able to figure out why.

Reply:
@ciremusyoka if we do that are there any `expect`s passing that look at the component props? If not then it can/will "pass" even though the plan mapped to props is `null` causing the mounted component to not render a map.